src/plugin.validation.dns.simply/Simply/SimplyDnsClient.cs

```diff
@@ -0,0 +1,100 @@
+﻿using PKISharp.WACS.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace PKISharp.WACS.Plugins.ValidationPlugins.Simply
+{
+    public class SimplyDnsClient
+    {
+        private readonly ILogService _log;
+        private readonly HttpClient _httpClient = new();
+        private readonly string _baseUrl = "https://api.simply.com/2";
+
+        public SimplyDnsClient(string account, string apiKey, ILogService log)
+        {
+            _log = log;
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", EncodeBasicAuth(account, apiKey));
+        }
+
+        public async Task CreateRecordAsync(string identifier, string domain, string value)
+        {
+            var objectId = GetObjectId(identifier);
+            var records = await GetRecordsAsync(objectId);
+
+            // Remove any dangling records
+            var existingRecords = records.Where(x => x.Type == "TXT" && x.GetHostname(objectId) == domain).ToList();
+            foreach (var record in existingRecords)
+            {
+                await DeleteRecordAsync(objectId, record.RecordId);
+            }

```

Comment:
I don't like this kind of code, it might delete records that are useful to the user. (Especially in DNS substitution scenarios).